x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_list.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { EuiBasicTable } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { Immutable } from '../../../../../common/endpoint/types';
+import { TrustedApp } from '../../../../../common/endpoint/types/trusted_apps';
+import { getTrustedAppsListPath } from '../../../common/routing';
+
+import {
+  getCurrentError,
+  getLastPresentData,
+  isInProgressBinding,
+} from '../state/async_data_binding';
+import { PageInfo } from '../state/items_page';
+import { ListViewState } from '../state/list_view_state';
+
+interface TrustedAppsListProps {
+  state: Immutable<ListViewState<TrustedApp>>;
+}
+
+export function TrustedAppsList(props: TrustedAppsListProps) {
+  const data = getLastPresentData(props.state.currentPage);
+  const history = useHistory();
+
+  return (
+    <EuiBasicTable
+      items={[...(data?.items || [])]}
```

Comment:
using a `selector` could avoid you having to do this here. the selector should always return an array

Reply:
Fixed